app/src/components/Modals/change-methodology.tsx

```diff
@@ -36,14 +26,63 @@ interface ChangeMethodologyProps {
   onClose: () => void;
   onChangeClicked: () => void;
   t: TFunction;
+  gpcReferenceNumber: string;
+  inventoryId: string;
 }
 
 const ChangeMethodology: FC<ChangeMethodologyProps> = ({
   isOpen,
   onClose,
   onChangeClicked,
   t,
+  gpcReferenceNumber,
+  inventoryId,
 }) => {
+  const toast = useToast();
+
+  const [deleteAllActivityValues, { isLoading }] =
```

Comment:
Would be nice to rename the state var to `isDeleteAllActivitiesLoading` or something like that for clarity and to prevent collisions...